File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/client/block/shape/BlockCollide.java
package pize.tests.voxelgame.client.block.shape;

import pize.physic.BoundingBox3;

public class BlockCollide{
    
    public static final BlockCollide SOLID = new BlockCollide(new BoundingBox3(0, 0, 0, 1, 1, 1));
    
    
    final BoundingBox3[] boxes;
    
    public BlockCollide(BoundingBox3... boxes){
        this.boxes = boxes;
    }
    
    public BoundingBox3[] getBoxes(){
        return boxes;
    }
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/client/block/shape/BlockCollide.kt
package pize.tests.voxelgame.client.block.shape

import pize.physic.BoundingBox3

class BlockCollide(vararg boxes: BoundingBox3) {
    val boxes: Array<BoundingBox3>

    init {
        this.boxes = boxes
    }

    companion object {
        val SOLID = BlockCollide(BoundingBox3(0.0, 0.0, 0.0, 1.0, 1.0, 1.0))
    }
}
